docs/use-the-network/run-a-network-server/run-a-network-server.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: run-a-network-server
+hide_title: true
+sidebar_label: Run a Network Server
+slug: /use-the-network/run-a-network-server
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Run a Network Server
+
+The Helium Network is unique in that it provides public wireless infrastructure
+for privately operated LoRaWAN Network Servers. Details about how Helium
+blockchain primitives interoperate with LoRaWAN routing can be read about
+[here](/lorawan-on-helium).
+
+In this section, we will provide a step-by-step guide on actually getting a
+LoRaWAN Network Server (LNS) on Helium up and running.
+
+- [buy an OUI](/use-the-network/buy-an-oui)
+- [run Console](/use-the-network/run-console)
+
+Before pursuing this configuration though, it is worth considering.
+
+**Do I need to own my own OUI and operate my own Console?**
+
+- If you are developing a few prototype devices, it might not be time to operate
+  a network server. Helium operates [Console](/use-the-network/console) where
+  you can easily manage devices for an organization without dealing with OUI
+  management.
+
+- An OUI costs $900 in Data Credits at a minimum ($100 for the OUI and $800 for
+  the smallest address slab), so if that cost if prohibitive, use the Helium
+  console.
```

Comment:
use Helium Console

Reply:
I'll do a slight variant, "Helium's Console" to indicate the one operated by us, rather than "Helium Console" the OSS Project? :shrug: